.github/publish.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Create and publish a Docker image
+
+on:
+  push:
+    branches: ['release']
```

Comment:
this should be the `development` branch.
we have no branch called `release` in this repo. 

You should also add:
```
pull_request:
    branches: [ development ]
```
    
  see example here:
  https://github.com/Ishangoai/Ishango_Assessments/blob/development/.github/workflows/publish.yml